### PR TITLE
Add testModifyUserCreateLight and provider

### PR DESCRIPTION
**Pull request by pwalczysko**
# What this PR does

Adds a test where light admin creates another light admin. The creator and created have always the same set of permissions. 4 sets of permissions are tested, which are matching the light admin doc and the permissions are easily read from https://github.com/openmicroscopy/openmicroscopy/pull/5101#issuecomment-281472022 and https://github.com/openmicroscopy/openmicroscopy/pull/5101#issuecomment-281655507

# Testing this PR

CI suffices

# Related reading

This PR is showing that the lightAdmin creates lightAdmin workflow has no apparent blocker.
See https://github.com/openmicroscopy/openmicroscopy/pull/5394, cc @will-moore 

Ready to review

### CODE REVIEW

**Comment by mtbc** [comment]
See https://ci.openmicroscopy.org/job/OMERO-DEV-merge-integration-java/lastCompletedBuild/testngreports/integration/LightAdminRolesTest/ for test results.

File: components/tools/OmeroJava/test/integration/LightAdminRolesTest.java [hunk 1]
```diff
@@ -1640,6 +1640,71 @@ public void testModifyUserCreate(boolean isPrivileged,
...
+            permissions.add(AdminPrivilegeWriteScriptRepo.value);
+            permissions.add(AdminPrivilegeDeleteScriptRepo.value);
+        } else if (lightAdminType.equals("Organizer")) {
+            permissions.add(AdminPrivilegeChgrp.value);
+            permissions.add(AdminPrivilegeChown.value);
+            permissions.add(AdminPrivilegeModifyGroup.value);
+            permissions.add(AdminPrivilegeModifyGroupMembership.value);
+            permissions.add(AdminPrivilegeDeleteOwned.value);
+            //permissions.add(AdminPriilegeDeleteManagedRepo.value);
+            //permissions.add(AdminPriilegeDeleteFile.value);
```
Comment (mtbc) [inline components/tools/OmeroJava/test/integration/LightAdminRolesTest.java]: add a "v" :smiley:

Comment (pwalczysko) [reply components/tools/OmeroJava/test/integration/LightAdminRolesTest.java]: @mtbc : Thank you

**Review by mtbc — APPROVED** [review]
Code looks as intended and passes in CI.

**Pull request merged**